src/unit/categories.js

```diff
@@ -0,0 +1,11 @@
+define(function() {
+
+/**
+ * categories()
+ *
+ * Return all unit categories.
+ */
+return [ "acceleration", "angle", "area", "duration", "length", "mass", "power", "pressure",
+"speed", "temperature", "volume" ];
```

Comment:
Modified this slightly to add "digital" category. Are there other places in the codebase where you're caching once based on the cldr data?